app/src/main/java/br/org/cesar/discordtime/stickysessions/data/remote/repository/SessionRemoteRepository.java

```diff
@@ -86,7 +86,7 @@ public SessionRemoteRepository(SessionService service, Mapper<Session, SessionRe
     }
 
     private boolean isSessionFromThisMeeting(Session session, String meetingId) {
-        return session.createdAt.equals(meetingId);
+        return session != null && session.createdAt != null && session.createdAt.equals(meetingId);
```

Comment:
could be simplified to 
`session != null && meetingId.equals(session.createdAt)`

Reply:
forget it. meetingId can also be null, so it would also be vulnerable to NPE.